src/tests/ftest/network/zero_config.py

```diff
@@ -87,7 +87,8 @@ def get_log_info(self, hosts, dev, env_state, log_file):
             bool: status of whether correct device was used.
 
         """
-        cmd = "head -50 {}".format(log_file)
+	# anticipate log switch
+        cmd = "if [ -e {}.old ]; then head -50 {}.old; else head -50 {}; fi".format(log_file)
```

Comment:
(pylint-line-too-long) Line too long (93/80)
(pylint-too-few-format-args) Not enough arguments for format string

Reply:
I don't think you can re-use a single format argument with indexing it:

```
$ python -c 'print("{} {} {}".format("foo"))'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
IndexError: tuple index out of range
$ python -c 'print("{0} {0} {0}".format("foo"))'
foo foo foo
```

```suggestion
        cmd = "if [ -f {0}.old ]; then head -50 {0}.old; else head -50 {0}; fi".format(log_file)
```

Reply:
oops, I thought index 0 would be the default, particularly when there is only one parameter !!... Python is sometimes surprisingly very "fussy" !!...